MovieQuizTests/ArrayTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ArrayTests.swift
+//  MovieQuizTests
+//
+//  Created by Артем Кохан on 04.11.2022.
+//
+
+import Foundation
+import XCTest
+
+@testable import MovieQuiz
+
+class ArrayTest: XCTestCase {
+    func testGetValueInRange() throws {
+//        Given
```

Comment:
Два таба перед, а не после: 
```suggestion
        // Given
```